README.md

```diff
@@ -103,3 +103,14 @@ Provides an easy to consume interface that binds native callback functions to br
 
 ## Language Bindings (todo)
 GBW is developed in C++, but will aim to provide bindings for other GTK supported languages.
+
```

Comment:
I know that the next line is a requirement from you sponsor, but the top level documentation should be less verbose than this, so omit it.